### DISCUSSION TITLE
Rounded corners if pannel lenght is < 100

### DISCUSSION THREAD

**Discussion by alxgvr** [home-sweet-gnome/dash-to-panel#1990] [Ideas]
Hi. I use panel length 90% and auto hide. So, I can close maximized windows without triggering appear action. 

Issue is - panel in this mode has ugly right angle corners (90) which are not fitting Gnome aesthetics. 
Please, add option to set how rounded corners should be. I want panel to have exact roundness as gnome extensions on panel. 
![Screenshot from 2023-11-16 23-22-52](https://github.com/home-sweet-gnome/dash-to-panel/assets/19569543/6e707d8d-6a55-4814-a2f4-0808c8265e52)

**Comment by bsg-univers** [comment]
Please add rounded corners in the compact view!
